src/Modal/Modal.module.scss

```diff
@@ -49,3 +58,25 @@
   color: var(--black) !important;
   border-color: #d9dce9 !important;
 }
+
+@keyframes fadeIn {
+  0% {
+    background: rgba(0, 0, 0, 0);
+  }
+
+  100% {
+    background: rgba(0, 0, 0, 0.7);
+  }
+}
+
+@keyframes moveDown {
+  0% {
+    transform: translateY(-1000px) translateX(-50%);
```

Comment:
In the animations much better to use similar property signatures, e.g. `transform: translate3d(-50%, -1000px, 0);`